### PR TITLE
Filter by language #66

### PR DESCRIPTION
**Pull request by lachatak**
language filter has been added.

my test code is available here:
https://github.com/lachatak/tweetsampler/tree/twitter4s
there are couple of things still missing so it is not merged to master.
I need 'replace' and 'shutdown' function before the demo can be merged to master and deployed to heroku for demonstration purpose.

### CODE REVIEW

**Comment by DanielaSfregola** [comment]
Awesome! Thank you very much for your contribution 👍 

LGTM, just one minor detail on the param name that I can change by myself after this PR is merged.

**Comment by DanielaSfregola** [comment]
I'll try to implement the filter and shutdown functions asap since you need them.

Cheers,
D.

File: src/main/scala/com/danielasfregola/twitter4s/http/clients/streaming/statuses/TwitterStatusClient.scala [hunk 1]
```diff
@@ -30,15 +31,19 @@ trait TwitterStatusClient extends TwitterStreamListenerHelper with StreamingOAut
...
+    * @param language : Empty by default. A comma separated list of 'BCP 47' language identifiers.
+    *                    For more information <a href="https://dev.twitter.com/streaming/overview/request-parameters#language" target="_blank">
+    *                      https://dev.twitter.com/streaming/overview/request-parameters#language</a>
     * @param stall_warnings : Default to false. Specifies whether stall warnings (`WarningMessage`) should be delivered as part of the updates.
     * @param f: the function that defines how to process the received messages
     */
   def getStatusesFilter(follow: Seq[Long] = Seq.empty,
                         track: Seq[String] = Seq.empty,
                         locations: Seq[Double] = Seq.empty,
+                        language: Seq[Language] = Seq.empty,
```
Comment (DanielaSfregola) [inline src/main/scala/com/danielasfregola/twitter4s/http/clients/streaming/statuses/TwitterStatusClient.scala]: In other endpoints I have already sometimes slightly changed the name of the params to improve the usability of the client, so a little diversion would not be the end of the world. At the end our goal is having a client that is super-easy to use ;)

I think going for a little diversion of using the plural here would be acceptable....what do you think @lachatak?

**Comment by lachatak** [comment]
plural languages parameter fixed

**Comment by DanielaSfregola** [comment]
LGTM

**Comment by DanielaSfregola** [comment]
Together with PR #72, this resolves #70

**Comment by DanielaSfregola** [comment]
This is going to be released in version 3.0

**Pull request merged**